.gitignore

```diff
@@ -3,19 +3,6 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
```

Comment:
@Sotatek-HauTran3 Please remove it because we deleted contents visual studio automatically generate